include/xlnt/utils/numeric.hpp

```diff
@@ -133,57 +130,17 @@ class number_serialiser
     }
 
 public:
-    explicit number_serialiser()
-        : should_convert_comma(localeconv()->decimal_point[0] == ',')
-    {
-    }
+    explicit number_serialiser() = default;
 
     // for printing to file.
     // This matches the output format of excel irrespective of current locale
-    std::string serialise(double d) const
-    {
-        char buf[30];
-        int len = snprintf(buf, sizeof(buf), "%.15g", d);
-        if (should_convert_comma)
-        {
-            convert_comma_to_pt(buf, len);
-        }
-        return std::string(buf, static_cast<size_t>(len));
-    }
+    XLNT_API std::string serialise(double d) const;
```

Comment:
Should we export this implementation detail?

Reply:
If I remember correctly, this is because of the unit tests, just like for the parsing functions themselves. But I have to check it out again.